Documentation/gettingstarted/host-firewall.rst

```diff
@@ -6,13 +6,19 @@
 
 .. _host_firewall:
 
-*************
-Host Firewall
-*************
+******************************************
+Host Firewall (beta when using kube-proxy)
+******************************************
 
 This document serves as an introduction to Cilium's host firewall, to enforce
 security policies for Kubernetes nodes.
 
+.. note::
+
+    The host firewall is a beta feature when running without our kube-proxy
```

Comment:
Maybe makes sense to add a note saying that it's in beta until both issues have been fixed (with the links to them)?